packages/@guardian/source-react-components-development-kitchen/components/divider/Divider.tsx

```diff
@@ -32,6 +33,51 @@ const decideSpace = (
 				: css`
 						margin-top: ${space[12]}px;
 				  `;
+		case 'none':
+			return css`
+				margin-top: 0;
+			`;
+	}
+};
+
+const decideSize = (size: Sizes, displayText?: string) => {
```

Comment:
Is this exposing a weakness with the original api? Passing in `displayText` and using that to decide size seems.... wrong? Not really sure what the right api would be to be honest and I appreciate this is out of scope but might be worth some thought

Reply:
I'd be interested in exploring the possibility of shipping the Divider with no margin at all.

Instead we could provide some exported css blocks like `dividerSpacingTight` (completely contrived name) which the end user could then apply at will to customise their Divider